src/index.js

```diff
@@ -0,0 +1,58 @@
+import navUl from './navbar';
+import homePage from './homePage';
+import displayMenu from './menu';
+import contactPage from './contact';
+
+const outerContainer = document.createElement('div');
+outerContainer.id = 'container';
```

Comment:
- [ ] It will be great to use the name given for this ID. Since this is the `div` that is the parent to your application, it should be given an id of `content` ⬇️ 
![append_requirement](https://user-images.githubusercontent.com/32361963/127464535-4095b525-fe65-48aa-b60e-628407cd26dd.png)